acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/signer/runner/Eth2SignerRunner.java

```diff
@@ -219,4 +205,23 @@ public int metricsPort() {
       return signerConfig.getMetricsPort();
     }
   }
+
+  private Path createTempDirectory(final String prefix) {
+    try {
+      final Path certificateDirectory = Files.createTempDirectory(prefix);
```

Comment:
nit: not necessarily the certdirectory - could be the datadir ... maybe rename to just "tempDir"?

Reply:
done